playbooks/roles/forum/tasks/deploy.yml

```diff
@@ -27,6 +27,47 @@
     - install
     - install:configuration
 
+- name: create the watchdog config
+  template:
+    src: watchdog.conf.j2
+    dest: "{{ supervisor_available_dir }}/watchdog.conf"
```

Comment:
Could we rename it to something like `forum_mongo_watchdog` or `forum-watchdog`? It's not clear what it watches in this directory.